client/src/components/QRCode.tsx

```diff
@@ -17,7 +17,7 @@ export const QRCode: React.FC<Props> = ({ invitationUrl, connectionState }) => {
   const isMobile = useMediaQuery({ query: '(max-width: 768px)' })
   const isLarge = useMediaQuery({ query: '(max-width: 1242px)' })
 
-  const isCompleted = connectionState === 'responded' || connectionState === 'complete'
+  const isCompleted = connectionState === 'responded' || connectionState === 'completed'
```

Comment:
Does this mean that the demo ONLY supports OOB or is this backwards compatible with rfc-160 connections?

Reply:
The demo still uses legacy invitations with the states from the didexchangeprotocol.